apps/autoscheduler/PerfectHashMap.h

```diff
@@ -53,14 +54,15 @@ class PerfectHashMap {
     }
 
     void upgrade_from_small_to_large(int n) {
-        internal_assert(occupied <= max_small_size) << occupied << " " << max_small_size << "\n";
+        // internal_assert(occupied <= max_small_size) << occupied << " " << max_small_size << "\n";
```

Comment:
Presumably these are commented out because we have a test now?

Reply:
Reinserted.